devicetypes/smartthings/zigbee-dimmer-power.src/zigbee-dimmer-power.groovy

```diff
@@ -18,15 +18,14 @@ metadata {
         capability "Configuration"
         capability "Refresh"
         capability "Power Meter"
-        capability "Sensor"
```

Comment:
I think we should leave the sensor capability here because this device can detect power usage.